cubedash/_stac.py

```diff
@@ -0,0 +1,235 @@
+import logging
+
+from collections import OrderedDict
+from flask import Blueprint, request, abort, url_for
+
+from datacube.api.query import Query
+from datacube.model import Range
+from datacube.utils.geometry import CRS, Geometry
+
+import datetime
+from dateutil.parser import parse
+from pandas import Timestamp, DateOffset
+import yaml
+import json
+import itertools
+
+from . import _utils as utils
+from . import _model
+
+
+_LOG = logging.getLogger(__name__)
+bp = Blueprint('stac', __name__, url_prefix='/stac')
+
+CFG = yaml.load(
+    """
+    homepage: "http://www.ga.gov.au/"
+
+    license:
+      name: "Creative Commons Attribution 4.0 International"
+      link: "https://spdx.org/licenses/CC-BY-4.0.html#licenseText"
+      short_name: "CC-BY-4.0"
+      copyright: "DEA, Geoscience Australia"
+
+    contact:
+      name: "Geoscience Australia"
+      organization: "Commonwealth of Australia"
+      email: "sales@ga.gov.au"
+      phone: "+61 2 6249 9966"
+      url: "http://www.ga.gov.au"
+    """
+)
+
+MAX_DATASETS = 100
+DATASETS_PER_REQUEST = 20
+
+
+@bp.route('/')
+def root():
+    return utils.as_json({})
```

Comment:
A blank response might be confusing to users -- it looks like a mistake, and it may confuse some stac clients.

A 404 error could more explicitly tell them that a root catalog is not support yet:
```suggestion
    abort(404, "Only /stac/search is currently supported")
```
(perhaps a 501-not-implemented error is more correct, but it's cacheable as a permanent response, which doesn't sound wise to me. All other unimplemented paths are 404s by default, so I'm leaning towards it.)